plugins/cache/bap_cache_gc.mli

```diff
@@ -0,0 +1,4 @@
+
+val shrink : ?threshold:int64 -> upto:int64 -> unit -> unit
```

Comment:
we always used megabytes to specify sizes in our user interface. Why we need to use bytes here?